frontend/src/lib/components/PropertiesTable/PropertiesTable.tsx

```diff
@@ -198,17 +198,20 @@ export function PropertiesTable({
     }
 
     const objectProperties = useMemo(() => {
-        if (!(properties instanceof Object)) {
+        if (!properties || !(properties instanceof Object)) {
```

Comment:
Isn't it the case that every instance of `Object` is truthy?

Reply:
You're right, I was thinking of `typeof null` which is `"object"`.

Reply:
Actually it'd be good to keep this, as TypeScript doesn't narrow `properties` which is `Record<string, any> | undefined` without it, so `Object.entries(properties)` gives a type error.